bridge-proto/proto/query.proto

```diff
@@ -226,6 +231,9 @@ message QueryParameters {
 
   // Whether to decorate the result set with comparable byte data and per-row paging states.
   bool enriched = 11;
+
+  // Configurable resume mode for page state creation; only used when enriched = true.
```

Comment:
Does not really explains everything, add:

* should reference that it's for the paging state creation for each row
* if not specified, that the paging state is not set then
* difference between resume modes, maybe at the enum declaration